Component/Attributes.php

```diff
@@ -147,7 +155,7 @@ protected function checkForAttributeUpdates($attributeCode, $attributeArray, $at
 
             $name = $this->mapAttributeConfig($name);
 
-            if ($name == 'option') {
+            if (in_array($name, $this->skipCheck)) {
```

Comment:
@zxon Is this component used for customer attribute updates?

Reply:
@paulpartington-cti The Customer Attributes component extends the Attributes component.

Reply:
@zxon cool